src/app/dashboard-view/dashboard-view.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from "@angular/core";
+
+import { Users } from "./user.model";
+
+@Component({
+  selector: "app-dashboard-view",
+  templateUrl: "./dashboard-view.component.html",
+  styleUrls: ["./dashboard-view.component.css"],
+})
+export class DashboardViewComponent implements OnInit {
+  users: Users[] = [
+    { firstName: "Mike", lastName: "Wheeler", id: 16850011 },
+    { firstName: "Dustin", lastName: "Henderson", id: 17469805 },
+    { firstName: "Lucas", lastName: "Sinclair", id: 15678360 },
+    { firstName: "Will", lastName: "Byers", id: 18273800 },
+  ];
```

Comment:
This array should be defined in a service rather than in this component. Please make this change.

Reply:
Since this branch just contains the template of the app and there is no interactivity, I have defined the array of users in this component. I have moved this to a service in the next PR and injected the service here. If required I will create the service in this branch. Please let me know if I should do the same.

Reply:
Service has been added.